apps/site/lib/site_web/templates/event/_event_list.html.eex

```diff
@@ -7,12 +7,18 @@
         range = %{start: event_teaser.date, stop: event_teaser.date_end}
         path_fn = fn -> cms_static_page_path(@conn, event_teaser.path) end
       %>
-      <li class="list-group-item">
-        <div>
-          <%= render_duration(range.start, range.stop) %>
+      <li class="list-group-item l-flex-container">
+        <div class="m-event__date-circle">
+          <small><%= format_month(range.start) %></small>
+          <h3 class="m-event__day"><%= format_day(range.start) %></h3>
```

Comment:
Why h3? I'd rather avoid using headers for these items..

Reply:
Was matching the ZeroHeight again.  Looks like I should start a thread in slack about our typography system